distribution/lib/Standard/Table/0.0.0-dev/src/Delimited/Delimited_Format.enso

```diff
@@ -14,7 +14,8 @@ type Delimited_Format
 
        If a row does not match the first row's column count, the function raises
        an `Invalid_Row`. If a quote is opened and never closed, a
-       `Mismatched_Quote` warning occurs.
+       `Mismatched_Quote` warning occurs. If an empty file is passed, an
+       `Empty_File_Error` is raised.
```

Comment:
Shall we put them on separate lines (maybe with a `-` list).
```suggestion
       `Mismatched_Quote` warning occurs. 
        If an empty file is passed, an `Empty_File_Error` is raised.
```

Reply:
Yeah, I think a list makes sense here.